Content.Server/Atmos/GasMixture.cs

```diff
@@ -96,6 +96,49 @@ public float GetMoles(Gas gas)
             return GetMoles((int)gas);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public float GetMoles(string gas)
+        {
+            return GetMoles(GetGasId(gas));
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public int GetGasId(string gasName)
+        {
+            int gas = -1;   //crash the game if you give a wrong gasName
+            switch(gasName)
+            {
```

Comment:
Whatever this is being used for should just use the gas enums instead

Reply:
In ``ReactionPrototype`` the reactants and products are organized in a string-ReactionSubjectPrototype dictionary, which makes the reaction logic and yml prototypes a whole lot simpler to write, as the same string can correspond to a reagent or a gas.